analyses/molecular-subtyping-LGAT/run_subtyping.sh

```diff
@@ -11,5 +11,6 @@ SUBSET=${OPENPBTA_SUBSET:-1}
 
 if [ "$SUBSET" -gt "0" ]; then
   Rscript -e "rmarkdown::render('01-subset-files-for-LGAT.Rmd')"
+  Rscript -e "rmarkdown::render('02-subset-fusion-files-LGAT.Rmd')"
```

Comment:
Note to myself when I review next week: This effectively means this module is not tested in CI at all - is there an alternative?

Reply:
Let me know how/if I should add the Part2-3 LGAT PRs outside the if condition. Just going with the condition for subset I'd put all the subsetting files there and would eventually add a final notebook to gather annotation outside this condition.